properties.ini

```diff
@@ -0,0 +1,157 @@
+; this file is used to define all necessary options for virt-who testing
+; all default values are null, need to update according to the test requirements
+; hypervisor-builder can help to get some hypervisor and guest information
+
+[virtwho]
+; host= is the rhel system to run virt-who service
+; repo= is the base url of the rhel compose
+host=
+username=
+password=
+port=
+repo=
+proxy_server=
+proxy_prot=
+
+[rhsm]
+server=
+username=
+password=
+prefix=
+port=
+default_org=
+
+[satellite]
+server=
+username=
+password=
+prefix=
+port=
+default_org=
+secondary_org=
+activation_key=
+
+[vcenter]
+; esx_ datas are for the esxi host which the guest is based on
+server=
+username=
+password=
+ssh_username=
+ssh_password=
+esx_ip=
+esx_username=
+esx_password=
+esx_uuid=
+esx_hwuuid=
+esx_hostname=
+esx_version=
+esx_state=
+esx_cpu=
+esx_cluster=
+guest_name=
+guest_ip=
+guest_username=
+guest_password=
+guest_uuid=
+guest_state=
+
+[xen]
+server=
+username=
+password=
+uuid=
+hostname=
+version=
+state=
+cpu=
+guest_name=
+guest_ip=
+guest_username=
+guest_password=
+guest_uuid=
+guest_state=
+
+[hyperv]
+server=
+username=
+password=
+uuid=
+hostname=
+version=
+state=
+cpu=
+guest_name=
+guest_ip=
+guest_username=
+guest_password=
+guest_uuid=
+guest_state=
+
+[rhevm]
+; vdsm_ datas are for the vdsm host which the guest is based on
+server=
+username=
+password=
+ssh_username=
+ssh_password=
+vdsm_ip=
+vdsm_username=
+vdsm_password=
```

Comment:
Think about for our test cases, are there some cases will use the ssh account to login rhevm server or vdsm host?

Reply:
The guest_stop/start/suspend/resume cases need the ssh rhevm access to run command.

In virtwho-ci some functions are defined with ssh_vdsm=None parameters and it seems that we never use it in cases, so I think we can remove it and if there will be requirement we can add it back.

Reply:
We only do the 'add/delete/start/stop/suspend/resume/migrate/clone' for vcenter environment, unless we have specific issues for other hypervisors.
What do you think?

Reply:
Agree with you. 
My previous idea was add all attributes that must be or maybe useful, and remove the useless ones at last.
But the better should be firstly just add the must necessary ones, and add others when have requirements, which will keep not redundent.